packages/blade/src/components/Chip/Chip.stories.tsx

```diff
@@ -0,0 +1,291 @@
+import type { ComponentStory, Meta } from '@storybook/react';
+import React from 'react';
+import { Code, Text, Title } from '../Typography';
+import type { ChipGroupProps } from './ChipGroup';
+import { ChipGroup as ChipGroupComponent } from './ChipGroup';
+import { Chip as ChipComponent } from './Chip';
+import {
+  PaymentLinksIcon,
+  TagIcon,
+  SmartphoneIcon,
+  ThumbsUpIcon,
+  ThumbsDownIcon,
+} from '~components/Icons';
+import { Sandbox } from '~utils/storybook/Sandbox';
+import StoryPageWrapper from '~utils/storybook/StoryPageWrapper';
+import type { BladeElementRef } from '~utils/types';
+import { Button } from '~components/Button';
+import { Box } from '~components/Box';
+import { getStyledPropsArgTypes } from '~components/Box/BaseBox/storybookArgTypes';
+import iconMap from '~components/Icons/iconMap';
+
```

Comment:
can you sort the story so chipgroup comes first in the navbar?
<img width="204" alt="Screenshot 2023-08-29 at 10 33 11 AM" src="https://github.com/razorpay/blade/assets/11384858/d132612c-8a3c-4b19-8f02-541dae0a49cc">

Reply:
Checking the docs https://storybook.js.org/docs/react/writing-stories/naming-components-and-hierarchy

Reply:
Unable to do it without compromising `Chip` folder position in the sidebar.